src/print.c

```diff
@@ -53,6 +54,20 @@ int apk_get_screen_width(void)
 	return apk_screen_width;
 }
 
+char apk_get_human_unit(size_t size, double *dest)
+{
+	char units[] = {'B', 'K', 'M', 'G', 'T'};
+	size_t i;
+	double s;
+
+	for (i = 0, s = size; s >= 1024 && i < sizeof(units); i++)
+		s /= 1024;
+
+	if (dest != NULL)
```

Comment:
Stylistically I nowadays like more something more concise in single line if possible:
`if (dest) *dest = s;`